lib/mix/tasks/phoenix.gen.channel.ex

```diff
@@ -52,6 +52,11 @@ defmodule Mix.Tasks.Phoenix.Gen.Channel do
     unless length(args) == 2 do
       raise_with_help
     end
+
+    args
```

Comment:
@josevalim does the namespace need to be considered?  In issue #921, even within the `Throwaway` namespace an error is still raised:

```elixir
(UndefinedFunctionError) undefined function: Throwaway.Repo.start_link/0
```

Apologizes in advance if I'm confused :smile:, thanks for your guidance.